src/components/Input.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+
+
+export const Input = ({variant, margin, handleChange, id, label, name, autoComplete }) => {
+    return (
+        <div>
```

Comment:
You don't need a ```div``` here, the ```TextField``` component already wraps all the children. Don't use divs until you really need it.